data/publiccloud/terraform/azure.tf

```diff
@@ -1,3 +1,12 @@
+terraform {
+  required_providers {
+    azurerm = {
+      version = "= 2.56.0"
```

Comment:
```
Initializing provider plugins...
- Finding latest version of hashicorp/random...
- Finding latest version of hashicorp/azurerm...
- Installing hashicorp/random v3.1.0...
- Installed hashicorp/random v3.1.0 (self-signed, key ID 34365D9472D7468F)
- Installing hashicorp/azurerm v2.57.0...
- Installed hashicorp/azurerm v2.57.0 (self-signed, key ID 34365D9472D7468F)
- ```
- I would also lock `random` module

Reply:
I still see this in your VR. 
```
Warning: Interpolation-only expressions are deprecated

  on plan.tf line 69, in resource "azurerm_resource_group" "openqa-group":
  69:             openqa_created_date = "${timestamp()}"
```

I think we need to solve this anyway . Or do you think that it is better to do this in separate PR ?

Reply:
Good ideas, I will do solve both.